src/main/target/STM32_UNIFIED/target.h

```diff
@@ -140,10 +152,16 @@
 #define TARGET_IO_PORTE 0xffff
 #define TARGET_IO_PORTF 0xffff
 
-#elif defined(STM32G47X)
+#elif defined(STM32G47X) || defined(STM32G47X_CC2500)
+#if defined(STM32G47X)
 #define TARGET_BOARD_IDENTIFIER "SG47"
 
 #define USBD_PRODUCT_STRING     "Betaflight STM32G47x"
+#else
+#define TARGET_BOARD_IDENTIFIER "SGCC"
```

Comment:
``SGCC`` is rather confusing for general ppl, I prefer ``SG4C`` for this case.

Reply:
I can change this to align with https://github.com/betaflight/betaflight/pull/11138.

Reply:
Done.